Documentation improvement (#63690)
Azure/azure-content: articles/app-service/app-service-key-vault-references.md

1. Create an [access policy in Key Vault](../key-vault/general/secure-your-key-vault.md#key-vault-access-policies) for the application identity you created earlier. Enable the "Get" secret permission on this policy. Do not configure the "authorized application" or `applicationId` settings, as this is not compatible with a managed identity.

    > [!NOTE]

>     > Key Vault references are not presently able to resolve secrets stored in a key vault with [network restrictions](../key-vault/general/overview-vnet-service-endpoints.md),       if you have a key vault with network restrictions your app service need to be hosted on an isolated [app service plan](https://docs.microsoft.com/it-it/azure/app-               service/overview-hosting-plans) and the key vault firewall needs to allow traffic coming from your hosting subnet.  

Reviewer: I think we can simplify this to:
```suggestion
    > Key Vault references are not presently able to resolve secrets stored in a key vault with [network restrictions](../key-vault/general/overview-vnet-service-endpoints.md) unless the app is hosted within an [App Service Environment](./environment/intro.md).
```

> | VaultName=_vaultName_;SecretName=_secretName_;SecretVersion=_secretVersion_ | The **VaultName** should the name of your Key Vault resource. The **SecretName** should be the name of the target secret. The **SecretVersion** should be the version of the secret to use. |

> [!NOTE] 

> > Versions are currently not required. If you are rotating secrets you should not include the version ( i.e. @Microsoft.KeyVault(SecretUri=https://myvault.vault.azure.net/secrets/mysecret)

Reviewer: The note on versions today is as desired. I'd ask that these changes be reverted.
